### PR TITLE
Carlo's Code Review

### PR DESCRIPTION
**Pull request by crugggiero**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ function App() {
   return (
     <div>
       <Map />
-      <SurfSpotCard />
+
```
Comment (bretthickman) [inline src/App.js]: Unused lines should be removed

**Review by bretthickman — COMMENTED** [review]
Other than one empty line, LGTM!

File: src/App.js [hunk 2]
```diff
@@ -1,22 +1,37 @@
...
 
 function App() {
   return (
-    <div>
-      <Map />
-      
-    </div>
+    <>
+      <Router>
+        <Navbar />
```
Comment (erikluu) [inline src/App.js]: New routing is clean. It makes sense. Just need to merge my pages with yours and we will be in tip top shape.

File: src/index.css [hunk 3]
```diff
@@ -13,9 +13,9 @@ code {
 }
 
 .map-container {
-  margin: auto;
-  height: 100vh;
-  width: 100vw;
+  height: 89vh;
```
Comment (erikluu) [inline src/index.css:16]: Essential!

File: src/components/Navbar.css [hunk 4]
```diff
@@ -0,0 +1,85 @@
+.navbar {
```
Comment (JackOrt) [inline src/components/Navbar.css]: LGTM

File: src/components/Navbar.js [hunk 5]
```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
```
Comment (JackOrt) [inline src/components/Navbar.js]: LGTM

**Pull request merged**